### PR TITLE
CIQUAL ericwang2

### PR DESCRIPTION
**Pull request by gnaweric**
CIQUAL PR

### CODE REVIEW

File: taxonomies/categories.txt [hunk 1]
```diff
@@ -25418,21 +25687,32 @@ it:Pont-l'évêque
...
+ciqual_food_code:en:12045
+ciqual_food_name:en:Reblochon cheese, from cow's milk
 
 <en:Sheep Cheeses
 <en:Soft cheeses with bloomy rind
 <en:French cheeses
 fr:Pérail
 
+<en:French cheeses
+en:Neufchâtel cheese from cow's milk
```
Comment (teolemon) [inline taxonomies/categories.txt:25711]: already exists somewhere else

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - fr:jambon-cuit-fume already is a synonym of jambons-blancs-fumes (en:cooked-ham-smoked) - cannot make a synonym of jambon-cuit-fume for en:cooked-ham-smoked
ERROR - fr:tomme already is a synonym of tommes (en:tomme-cheese-reduced-fat-around-13-fat) - cannot make a synonym of tomme for en:tomme-cheese-reduced-fat-around-13-fat
ERROR - fr:barre-au-chocolat already is a synonym of barres-chocolatees (en:coated-chocolate-bar-without-biscuit) - cannot make a synonym of barre-au-chocolat for en:coated-chocolate-bar-without-biscuit
ERROR - en:wafer-cookie already is a synonym of wafer-biscuit (en:wafer-cookie) - cannot make a synonym of wafer-cookie for en:wafer-cookie
ERROR - fr:tartelette-au-chocolat already is a synonym of tartelettes-au-chocolat (en:biscuit-small-tart-with-chocolate-pre-packed) - cannot make a synonym of tartelette-au-chocolat for en:biscuit-small-tart-with-chocolate-pre-packed

File: taxonomies/categories.txt [hunk 2]
```diff
@@ -4747,10 +4755,19 @@ ru:Кофе растворимый сублимированный, Раство
 
 <en:Instant beverages
 <en:Plant-based beverages
-en:Instant chicory
+en:Instant chicory with water
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:4758]: we should keep "Instant chicory" : that's what the product is.
it's fine to associate the CIQUAL category to it.

File: taxonomies/categories.txt [hunk 3]
```diff
@@ -20908,6 +20980,26 @@ wikipedia:en:https://en.wikipedia.org/wiki/Crouton
 # ingredient/fr:croûtons has 107 products in 5 languages @2019-04-21
 # category/croutons has 166 products @2019-05-19
 # usage:en:croutons (wheat flour, vegetable fat, edible salt, maltodextrin, grape sugar, onions, herbs, yeast extract, dehydrated garlic, flavoring, yeast, spice extract)
+ciqual_food_code:en:7430
+ciqual_food_name:en:Croutons
+
+<en:Breads
+en:Croutons
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:20987]: there's already a "en:Croutons" category above, we can't have a category with the same name. 
I have no idea what a "Croûtons à tartiner" is though :)     Les pains suédois ?

File: taxonomies/categories.txt [hunk 4]
```diff
@@ -21922,6 +22022,12 @@ nl:Amandelchocoladen
 ru:Шоколад с миндалём
 se:Choklad med mandlar, choklad med mandel
 
+<en:Chocolates
+en:Toblerone milk cholcolate bar w nougat
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:22026]: We shouldn't put brands or abbreviations (like "w").
I suggest "Milk chocolate bar with nougat"  --> notice I fixed the Ciqual error in chocolate :)

Comment (stephanegigandet) [reply taxonomies/categories.txt:22026]: and I forgot : all our categories, in French and in English, should be in plural

File: taxonomies/categories.txt [hunk 5]
```diff
@@ -25007,21 +25154,29 @@ wikidata:en:Q746471
...
-fr:Fromages à pâte persillée français, pâtes à moisissure interne français, bleus français
+fr:Fromages à pâte persillée français, pâtes à moisissure interne français, bleus français, Fromage bleu au lait de vache
 nl:Franse blauwschimmelkazen
+ciqual_food_code:en:12520
+ciqual_food_name:en:Blue cheese, from cow's milk
 
 <en:French blue-veined cheeses
 <en:Cow cheeses
 fr:Bleus d'Auvergne
+en:Auvergne blue cheese from cow's milk
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:25165]: just "Auvergne blue cheese"

File: taxonomies/categories.txt [hunk 6]
```diff
@@ -25007,21 +25154,29 @@ wikidata:en:Q746471
...
 de:Bleu d'Auvergne
 nl:Bleus d'Auvergne
 wikidata:en:Q883954
 country:en:France
+ciqual_food_code:en:12521
+ciqual_food_name:en:Auvergne blue cheese, from cow's milk
 
 <en:French blue-veined cheeses
 fr:Bleus de Gex, bleus du Haut-Jura, bleus de Septmoncel
```
Comment (stephanegigandet) [inline taxonomies/categories.txt:25174]: We should make 3 categories, one for each cheeese.  Also if there is an en: name, it needs to be first, before the fr: name

**Comment by teolemon** [comment]
Errors in the categories taxonomy definition:
ERROR - en:cheese-from-ewe-s-whey has an undefined parent 
ERROR - en:hard-cheese-emmental-type-cheese-reduced-fat has an undefined parent 
ERROR - en:plain-wafer-biscuit has an undefined parent 
ERROR - en:pork-belly-raw has an undefined parent 
ERROR - en:vanilla-flavoured-wafer-biscuit has an undefined parent 
ERROR - en:yogurt-greek-style-ewe-s-milk has an undefined parent 
ERROR - en:dry-spicy-pork-sausage-with-red-pepper has an undefined parent 
ERROR - en:spicy-pork-sausage-with-red-pepper-in-large-slices has an undefined parent

File: taxonomies/categories.txt (hunk 2 above)
Comment (teolemon) [inline taxonomies/categories.txt:4758]:
```suggestion
en:Instant chicory
```

File: taxonomies/categories.txt (hunk 3 above)
Comment (teolemon) [reply taxonomies/categories.txt:20987]: mmh…two different ids though

File: taxonomies/categories.txt (hunk 4 above)
Comment (teolemon) [inline taxonomies/categories.txt:22026]:
```suggestion
en:Milk chocolate bar with nougat, Toblerone
```

Comment (teolemon) [reply taxonomies/categories.txt:22026]: I added Toblerone as a synonym. Remove completely ?

File: taxonomies/categories.txt [hunk 7]
```diff
@@ -25322,14 +25503,20 @@ wikidata:en:Q2940381
 <en:French blue-veined cheeses
 <en:French cheeses
 fr:Fourme d'Ambert
+en:Fourme d'Ambert cheese from cow's milk
```
Comment (moon-rabbitOFF) [inline taxonomies/categories.txt:25506]:
```suggestion
en:Fourme d'Ambert cheese
```

**Pull request merged**